auraescript/aurae.ts

```diff
@@ -0,0 +1,65 @@
+export function print(...values: any[]) {
+    // @ts-ignore
+    Deno.core.print(values.map(toString).join(' ') + "\n");
+}
+
+function toString(value: any): string {
+    if (
+        typeof value === 'object' &&
+        !Array.isArray(value) &&
+        value !== null
+    ) {
+        return JSON.stringify(value, null, 2);
+    } else {
+        return value?.toString();
+    }
+}
+
+type CreateClientDefault = {
+    kind: "default";
+};
+type CreateClientPath = {
+    kind: "path";
+    path: string;
+};
+type CreateClientOpts = {
+    kind: "opts";
+    ca_crt: string;
+    client_crt: string;
+    client_key: string;
+    socket: string;
+};
+type CreateClient =
+    | CreateClientDefault
+    | CreateClientPath
+    | CreateClientOpts;
+
+export function createClient(opts: CreateClient = { kind: "default" }): AuraeClient {
+    if (typeof opts.kind === "undefined") {
+        // resolve kind
+        if ("path" in opts) {
+            opts.kind = "path";
+        } else if ("ca_crt" in opts) {
+            opts.kind = "opts";
+        }
+    }
+    let config;
+    switch(opts.kind) {
+        case "default":
```

Comment:
Linter is asking that the case/default blocks be wrapped in `{...}`

Reply:
I had to keep the `break;` or the linter would complain about the last line not being reachable